display_information/dictionary-renderer-graphics-overlay/README.md

```diff
@@ -1,41 +1,34 @@
-# Dictionary Renderer with Graphics Overlay
+# Dictionary renderer with graphics overlay
 
-Display mil2525d symbols.
+Create graphics using a local mil2525d style file and an XML file with key/value pairs for each graphic.
 
-The dictionary renderer creates these graphics using a local mil2525d style file and a XML file with key, value attributes for each graphic.
+![Dictionary Renderer Graphics Overlay Sample](dictionary-renderer-graphics-overlay.png)
 
-![](DictionaryRendererGraphicsOverlay.png)
+## Use case
+
+Use a dictionary renderer on a graphics overlay to display more transient data, such as military messages coming through a local tactical network.
+
+## How to use the sample
+
+When the sample is started, a dictionary renderer is applied and contents of a symbol style file are shown.
```

Comment:
This should be more of a how to use the sample rather than what it's doing: 

"Pan and zoom to see military symbols within the map"